nl.json

```diff
@@ -90,7 +95,7 @@
             "error-too-long" : "Dit duurt langer dan gedacht"
         },
         "shop" : {
-            "title" : "Huidige winkel",
+           "title" : "Huidige winkel",
```

Comment:
The spacing was correct before. Please don't remove the space before the translation key.